Ports/SDL2/patches/0001-Add-SerenityOS-platform-support.patch

```diff
@@ -15,6 +18,7 @@ Co-Authored-By: Kenneth Myhra <kennethmyhra@gmail.com>
 Co-Authored-By: Stephan Unverwerth <s.unverwerth@gmx.de>
 Co-Authored-By: Tim Schumacher <timschumi@gmx.de>
 Co-Authored-By: circl <circl.lastname@gmail.com>
+Co-Authored-By: kleines FilmrÃllchen <filmroellchen@serenityos.org>
```

Comment:
Is this actually encoded correctly? I'd imagine that GitHub would have no issue with displaying UTF-8.